source/_components/sensor.nut.markdown

```diff
@@ -0,0 +1,95 @@
+---
+layout: page
+title: NUT Sensor
+description: "Instructions on how to set up NUT sensors within Home Assistant."
+date: 2016-11-23
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: nut.png
+ha_category: Sensor
+---
+
+The `nut` sensor platform allows you to monitor a UPS (battery backup) by using data from the NUT server.
+
+To use this sensor platform, you need to add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: nut
+    name: UPS Name
+    host: 192.168.11.5
+    alias: ups_name
+    username: user
+    password: pass
+    resources:
+      - ups.load
+      - ups.realpower.nominal
+      - input.voltage
+      - battery.runtime
+```
```

Comment:
We try to keep the configuration sample minimal thus only show the required variables. This helps people to get started quickly by copy-&-paste the information without the hassle of thinking the optional stuff.